packages/terra-form-select/src/combobox/Menu.jsx

```diff
@@ -296,12 +296,16 @@ class Menu extends React.Component {
     if (keyCode === KeyCode.KEY_UP) {
       this.clearScrollTimeout();
       this.scrollTimeout = setTimeout(this.clearScrollTimeout, 500);
-      this.setState({ active: MenuUtil.findPrevious(children, active) });
+      this.setState({ active: MenuUtil.findPrevious(children, active) }, () => {
+        this.scrollIntoView();
+      });
       this.updateCurrentActiveScreenReader();
     } else if (keyCode === KeyCode.KEY_DOWN) {
       this.clearScrollTimeout();
       this.scrollTimeout = setTimeout(this.clearScrollTimeout, 500);
-      this.setState({ active: MenuUtil.findNext(children, active) });
+      this.setState({ active: MenuUtil.findNext(children, active) }, () => {
```

Comment:
The callback function can be passed without an anonymous call. 

```jsx
this.setState({}, this.scrollIntoView)
```

Reply:
Updated here c5d5a69bb0c00408c9a02f2abaed7a5b5050c938